gamearea.py

```diff
@@ -501,3 +549,26 @@ def start_timeout(self, time, callback=None, force=False):
 
     def redraw(self):
         GObject.idle_add(self.queue_draw)
+    
+    def save_highscore(self):
+        file_path = os.path.join(get_activity_root(), 'data', 'highscore')
+        highscore = [0]
+        if os.path.exists(file_path):
+            with open(file_path, "r") as fp:
+                highscore = fp.readlines()
+
+        int_highscore = int(highscore[0])
+        if not int_highscore > self.score:
+            with open(file_path, "w") as fp:
+                fp.write(str(self.score))
+
+    def load_highscore(self):
+        file_path = os.path.join(get_activity_root(), 'data', 'highscore')
+        if os.path.exists(file_path):
+            try:
+                with open(file_path, "r") as fp:
+                    highscore = fp.readlines()
+                return int(highscore[0])
+            except ValueError or IndexError:
```

Comment:
Correct me if I am wrong, `or` operator doesn't work for exceptions, innit?
```python
Python 3.9.6 (default, Jun 30 2021, 10:22:16)
[GCC 11.1.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> try:
...     raise KeyError
... except IndexError or KeyError:
...     print("caught!")
...
Traceback (most recent call last):
  File "<stdin>", line 2, in <module>
KeyError
```

At least, it doesn't work for me on Python 3.9.6

Reply:
right, I'll fix it

Reply:
Although it can be caught by supplying the exceptions as a tuple.
```python3
>>> try:
...     raise KeyError
... except (IndexError, KeyError):
...     print("caught!")
... 
caught!
```